.github/workflows/ci.yml

```
@@ -34,7 +34,7 @@ jobs:
       - name: Cache build artifacts
         uses: Swatinem/rust-cache@v2
       - name: cargo build
-        run: cargo build --all-targets
+        run: cargo build --all-targets --all-features
       - name: cargo test
         run: cargo test --all-targets
```

```suggestion
        run: cargo test --all-targets
        run: cargo test --all-targets --all-features
```
If we ever add tests behind feature flags, let's make sure we'll run them.

---

I cannot add this as a suggestion, but basically the same change for clippy:
```
diff --git a/.github/workflows/ci.yml b/.github/workflows/ci.yml
index 09f632c..10eb78a 100644
--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -56,4 +56,4 @@ jobs:
       - name: cargo fmt (check)
         run: cargo fmt -- --check -l
       - name: cargo clippy (warnings)
-        run: cargo clippy --all-targets -- -D warnings
+        run: cargo clippy --all-targets --all-feature -- -D warnings
```